static/js/ui.js

```diff
@@ -472,9 +472,13 @@ $(function () {
         timerender.set_full_datetime(message, time_elem);
     });
 
-    $('#streams_inline_cog').tooltip({ placement: 'left',
+    $('#streams_header h4').tooltip({ placement: 'right',
```

Comment:
Any particular reason why this code is here. Removing 

    $('#streams_header h4').tooltip({ placement: 'right',
                                       animation: false });

doesn't affects UI.